src/emqx_ct_helpers_jobs.erl

```diff
@@ -81,7 +81,15 @@
 	JobsConfig = [ { job, CurrentJob }, { all_jobs, Jobs } ],
 	ConfigUpdated = [ { ?JOBS_MATRIX_CONFIG, JobsConfig } | Config ],
 	case is_job_node() of
-		true -> override_function(FuncExists, Suite, ?CT_INIT_PER_SUITE, [ConfigUpdated], Config);
+		true ->
+			try override_function(FuncExists, Suite, ?CT_INIT_PER_SUITE, [ConfigUpdated], Config) of
```

Comment:
move it to a function ?

Reply:
The override_function already has a try catch for better exception reporting, but this specific implementation is due to needing to end_per_job to trigger despite any failure in init_per_suite, as such is unique.

Reply:
yes, I'll expand on the existing try catch in the override_function